fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -78,7 +78,7 @@ def take_off
         tool_name = ARGV.first ? ARGV.first.downcase : nil
 
         tool_name = process_emojis(tool_name)
-        tool_name = map_aliased_tools(tool_name)
+        tool_name = map_aliased_tools(tool_name) unless tool_name.nil?
```

Comment:
Thanks for investigation and quick fix @crazymanish
I propose moving the fix to `map_aliased_tools` (to return statement in line 145) to be consistent with `process_emojis` implementation by using safe navigation operator:
```ruby
def map_aliased_tools(tool_name)
    map = { ... }
    return map[tool_name&.to_sym] || tool_name
end
```
Another way could be grouping both`process_emojis` and `map_aliased_tools` in an unless statement `unless tool_name.nil?`, but as later we can see a check `if tool_name && ...` I would probably go for the former.

Reply:
Thank you very much for this suggestion. i applied the suggestion and learn about the `&.` operator 🙂

Reply:
I learnt about it during the review too ;)